templates/elasticsearch.yml.j2

```diff
@@ -125,7 +125,9 @@ http.enabled: false
 
 ################################### Gateway ###################################
 
+{% if elasticsearch_version < '2.0' %}
```

Comment:
I found the comparison with a string a bit weird, so I did a quick google search and stumbled on the `version_compare` filter. Maybe that would be more appropriate?

```
{% if elasticsearch_version | version_compare('2.0', '<') %}
```

See [Jinja2 filters in the Ansible documentation][1].

[1]: http://docs.ansible.com/ansible/playbooks_filters.html#version-comparison-filters